importer/main.py

```diff
@@ -139,12 +139,12 @@ def create_user_resources(user_id, user):
     logging.info("Creating user resources")
     # generate uuids
     if len(str(user[4]).strip()) == 0:
```

Comment:
please refactor all the code in this function by destructuring user and removing references by index.

eg.
```python
a, b, c, d = user
```

Reply:
Done